### DISCUSSION TITLE
Amazon Music

### DISCUSSION THREAD

**Discussion by gpiroddi** [rern/rAudio#1286] [Feature Requests]
Hallo,
is there a way to listen to Amazon Music HD with rAudio?
Thanks
Giovanni

**Comment by cYCL157** [comment]
i have looked for something from amazon in the past and didn't see something -- was hoping that if I could find something that I could get to work, maybe i could see how feasible it was for rern to integrate to.  If you know of a player, i have enough interest that I might be willing to dig into it some.
